### PR TITLE
Improvement of examples

### PR DESCRIPTION
**Pull request by Tim-ats-d**
* Improve code quality.
* Add line feed between logical structures.
* Rename variables and functions in `snake_case` instead of `PascalCase` according to [PEP8](https://www.python.org/dev/peps/pep-0008/).

### CODE REVIEW

**Comment by Tim-ats-d** [comment]
Squash my several commit in one.

File: example/fx.py [hunk 1]
```diff
@@ -59,24 +62,26 @@ def __init__(self, leds):
...
     def update(self, frame_time, fx):
         if self.empty:
             return
+
         self.env.time = frame_time
         for key in self.colors:
             self.colors[key] = ColorRgb.from_vec3(
-                *fx(self.env, self.leds[key][0], self.leds[key][1])).rgb
+                *fx(self.env,
+                    *self.leds[key][:2])).rgb
```
Comment (intrueder) [inline example/fx.py:78]: I cannot agree that this is an improvement. I would like to see in this place an explicit passing of the parameters x and y, no need to make code shorter in this case.

File: example/fx.py [hunk 2]
```diff
@@ -59,24 +62,26 @@ def __init__(self, leds):
                 lambda acc, pt: (max(acc[0], pt[0]), max(acc[1], pt[1])),
                 leds.values(), (float('-inf'), float('-inf')))
             self.env = FxEnv(Resolution(max_by_x, max_by_y))
-            self.colors = {x: (0, 0, 0) for x in leds}
+            self.colors = dict.fromkeys(leds, (0, 0, 0))
```
Comment (intrueder) [inline example/fx.py:65]: Nice 👍

**Review by intrueder — CHANGES_REQUESTED** [review]

File: example/fx.py (hunk 1 above)
Comment (Tim-ats-d) [reply example/fx.py]: Fixed.

**Review by intrueder — APPROVED** [review]
Looks good to me

**Pull request merged**